client/core/trade.go

```diff
@@ -457,6 +460,7 @@ func (t *trackedTrade) isSwappable(match *matchTracker) bool {
 func (t *trackedTrade) isRedeemable(match *matchTracker) bool {
 	dbMatch, metaData, proof, _ := match.parts()
 	if match.failErr != nil || proof.RefundCoin != nil {
+		log.Debugf("failErr = %v, RefundCoin = %v", match.failErr, proof.RefundCoin)
```

Comment:
Was showing `[DBG] CORE: failErr = <nil>, RefundCoin = ` indicating `proof.RefundCoin != nil`, an empty but non-nil slice although it was `nil` when stored to the client DB.  More logging == better.

With `spew.Dump` of the `db.MetaMatch`es, I noticed many other fields, such as signatures and the secret, that were not nil but empty as well.